code/implementation/docker/common/src/main/java/io/cattle/platform/docker/constants/DockerStoragePoolConstants.java

```diff
@@ -0,0 +1,8 @@
+package io.cattle.platform.docker.constants;
+
+public class DockerStoragePoolConstants {
+
+    public static final String DOCKER_KIND = "docker";
+    public static final String DOCKER_FORMAT = "docker";
```

Comment:
This was a refactor of fields used before. Was told by darren that he had them as two for a reason.